cpp/src/Slice/Parser.cpp

```diff
@@ -405,19 +405,6 @@ Slice::Type::usesClasses() const
 // Builtin
 // ----------------------------------------------------------------------
 
```

Comment:
`typeId` is only used in error messages in two places.
Both places didn't care about this `Ice::` prefix either.
I inlined the logic in the 2 places where it was needed.